README.md

```diff
@@ -148,7 +148,10 @@ FreeBSD etc). However, mlpack is in the repositories of many Linux distributions
 and so it may be easier to use the package manager for your system.  For example, 
 on Ubuntu, you can install mlpack with the following command:
 
-    $ sudo apt-get install libmlpack-dev
+    $ sudo apt-get install mlpack-bin
```

Comment:
We probably want users to install headers too, so maybe `sudo apt-get install libmlpack-dev mlpack-bin`?  We could also change the paragraph above to read `you can install the mlpack header files and command line executables with the following command:`.

Reply:
Ya that makes sense. Ill change this and upload new patch.